profile/README.md

```diff
@@ -1,3 +1,30 @@
 # California Cropland Monitoring and Modeling Framework
 
 ## About
+
+The California Cropland Monitoring and Modeling Framework (CCMMF) will be a biogeochemical model and data integration pipeline to generate inventories and future projections of soil carbon (SOC) stocks and greenhouse gas fluxes (GHG) from California’s croplands. 
+CCMMF will leverage Bayesian techniques to combine heterogeneous datasets into unified wall-to-wall inventories and climate and scenario-based projections. 
+Data layers will be generated as ensemble projections to facilitate propagation of uncertainty through downstream applications.
+
+CCMMF will produce new gridded time series of key agronomic practices including planting, harvest, irrigation, and tillage derived from satellite 
+measurements and agronomic statistics. 
```

Comment:
"CCMF will produce new gridded time-series information on ..... at a consistent spatial and temporal resolution of XX and monthly periods, respectively"? 

And this doesnt seem to capture the role of the model as well...perhaps "by combining remotely sensed data, agrinomic statistics, and biogeochemical modeling" or something like that?

Reply:
```suggestion
```